testing-dropwizard-4/pom.xml

```diff
@@ -0,0 +1,122 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>uk.gov.service.payments</groupId>
+        <artifactId>pay-java-commons</artifactId>
+        <version>1.0.0</version>
+    </parent>
+
+    <artifactId>testing-dropwizard-4</artifactId>
+    <name>Testing Utils DW 4</name>
+    <packaging>jar</packaging>
+
+    <properties>
+        <pact.version>4.0.10</pact.version>
```

Comment:
Does this version need bringing up to date (not necessarily in this pull request)? I see in Connector and Webhooks we're using `<pact.version>4.6.16</pact.version>` - and [that it usually gets updated in those repos by Dependabot](https://github.com/alphagov/pay-connector/commit/d6a50f6b1ca83b9d1e61d7d6787216241264b097#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R13). But in Adminusers it's still on v3.x. Any ideas what's going on there?

Reply:
In adminusers it's an unused version property, it's actually using:
`<groupId>au.com.dius.pact.provider</groupId>
  <artifactId>maven</artifactId>
  <version>4.6.16</version>`

Reply:
I'll test locally if we can upgrade pay-java-commons to 4.6.16

Reply:
Ahh, ok I got it 🤦 
In other projects we use the provider and/or consumer framework 4.6.16). Here we have the jvm framework (latest 4.0.10)

`<groupId>au.com.dius.pact.consumer</groupId>`
  `<artifactId>junit</artifactId>`
vs
`<groupId>au.com.dius</groupId>`
 `<artifactId>pact-jvm-provider-junit</artifactId>`

Reply:
Aha, that makes sense. Thanks for checking @SandorArpa!